frontend/components/Header/Nav/NavList.tsx

```diff
@@ -79,7 +103,9 @@ const NavList = ({ navItems, setMenuOpen }: NavListProps) => {
     <StyledNavMenuContainer>
       <p>
         <span>Menu</span>
-        <StyledCloseIcon onClick={() => setMenuOpen(false)} />
+        <StyledCloseIconWrapper>
+          <StyledCloseIcon onClick={() => setMenuOpen(false)} />
```

Comment:
```suggestion
        <StyledCloseIconWrapper onClick={() => setMenuOpen(false)}>
          <StyledCloseIcon />
```
if you tab to the button and hit space then it doesn't trigger the close action.

It might also be worth renaming StyledCloseIconWrapper to StyledCloseButton, and moving the 
```
  height: 40px;
  width: 40px;
```
inside the styling for the button. That would make this bit look like:
```suggestion
        <StyledCloseButton onClick={() => setMenuOpen(false)} >
          <CloseIcon />
```